dynamodump/dynamodump.py

```diff
@@ -33,6 +33,10 @@
 BATCH_WRITE_SLEEP_INTERVAL = 0.15  # seconds
 DATA_DIR = "data"
 DATA_DUMP = "dump"
+
```

Comment:
Please remove the additional new lines and sort them alphabetically

Reply:
done